ports/stm32/Makefile

```diff
@@ -698,10 +688,10 @@ $(GEN_PLLFREQTABLE_HDR): $(PLLVALUES) | $(HEADER_BUILD)
 
 $(BUILD)/modstm.o: $(GEN_STMCONST_HDR)
 # Use a pattern rule here so that make will only call make-stmconst.py once to
-# make both modstm_const.h and modstm_qstr.h
-$(HEADER_BUILD)/%_const.h $(BUILD)/%_qstr.h: $(CMSIS_MCU_HDR) make-stmconst.py | $(HEADER_BUILD)
+# make both modstm_const.h
+$(HEADER_BUILD)/%_const.h: $(CMSIS_MCU_HDR) make-stmconst.py | $(HEADER_BUILD)
```

Comment:
This comment about "both" is no longer valid, and maybe this rule can just become:
```
$(HEADER_BUILD)/modstm_const.h: $(CMSIS_MCU_HDR) make-stmconst.py | $(HEADER_BUILD)
```